app/frontend/src/components/forms/submission/MySubmissionsTable.vue

```diff
@@ -4,7 +4,8 @@
       <v-row class="mt-6" no-gutters>
         <v-col class="text-center" cols="12" sm="10" offset-sm="1">
           <h1>Previous Submissions</h1>
-          <h2>{{ form.name }}</h2>
+          <h2 v-if="formId">{{ form.name }}</h2>
+          <h2 v-else>All Forms</h2>
```

Comment:
Should this be in the other format?
```html
<h2>{{ formId ? form.name : 'All Forms' }}</h2>
```

Reply:
I don't know, stylistically I kind of like the v-if dom element style rather then ternaries in expressions. We probably mix the 2 all over the place. what do you think is cleaner?

Reply:
Fixed